domains/kirill-adamuk.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "superhorsy",
+        "email": "adamuk.kirill@gmail.com"
+    },
+    "record": {
+        "CNAME": "superhorsy.github.io"
```

Comment:
I cannot reach your site (404 page)

Reply:
@Stef-00012  Hi there! I have my website available at superhorsy.github.io/blog, but I think I can not set CNAME for such domain, what should I do in that case?  I choose CNAME because doc says so: https://www.is-a.dev/docs/github_pages/#using-is-adev-for-all-github-pages-sites-in-your-account

Reply:
Sorry, didn't read docs carefully, the website is available now, please check again.